src/main/scala/org/globalforestwatch/summarystats/treecoverloss/TreeLossSummary.scala

```diff
@@ -13,30 +13,30 @@ import scala.annotation.tailrec
 
 /** LossData Summary by year */
 case class TreeLossSummary(stats: Map[TreeLossDataGroup, TreeLossData] =
-                           Map.empty,
-                           kwargs: Map[String, Any])
+                           Map.empty)
   extends Summary[TreeLossSummary] {
 
   /** Combine two Maps and combine their LossData when a year is present in both */
   def merge(other: TreeLossSummary): TreeLossSummary = {
     // the years.combine method uses LossData.lossDataSemigroup instance to perform per value combine on the map
-    TreeLossSummary(stats.combine(other.stats), kwargs)
+    TreeLossSummary(stats.combine(other.stats))
   }
 }
 
 object TreeLossSummary {
   // TreeLossSummary form Raster[TreeLossTile] -- cell types may not be the same
 
-  implicit val mdhCellRegisterForTreeLossRaster1
-  : CellVisitor[Raster[TreeLossTile], TreeLossSummary] =
-    new CellVisitor[Raster[TreeLossTile], TreeLossSummary] {
+  def getGridVisitor(kwargs: Map[String, Any]): GridVisitor[Raster[TreeLossTile], TreeLossSummary] =
```

Comment:
Codacy found an issue: [Method is longer than 50 lines](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=6292950)